apsuite/orbcorr/orbit_correction.py

```diff
@@ -262,6 +261,8 @@ def set_delta_kicks(self, dkicks):
     def set_kicks(self, kicks):
```

Comment:
Also, shouldn't we be working with the `set_delta_kicks` method in `correct_orbit`? I believe the `set_kicks` is incorrect since the pseudo-inversion is giving us deltas, right?